bids/grabbids/bids_validator.py

```diff
@@ -1,4 +1,8 @@
+from copy import deepcopy
```

Comment:
If you just need to copy a dictionary shallowly, you should be able to use `dict.copy()`, without having to import from `copy`.